### PR TITLE
Update registration_runbook.md

### PR DESCRIPTION
**Pull request by a4orce84**
Updating runbook with 2023 lessons learned and updates.

### CODE REVIEW

**Review by a4orce84 — COMMENTED** [review]
Made updates to registration runbook with lessons learned from 2023.

File: registration_runbook.md [hunk 1]
```diff
@@ -1,57 +1,42 @@
...
+This runbook is focused on Registration / Ateendees and what you need to prep for taking on this role! There are a several key elements that are needed to be done before, during, and after the conference to ensure a succesful completion of the Registration role. In a nutshell, our job is to ensure that attendees can register, attend, and are reminded of the awesome DevOps exerpeince they will experience at DevOpsDays Austin. Please read below to learn more!
 
 ## Role description
 
-What does this role do? What are you primarily responsible for?
+The role of registration is centered around the idea of selling / supplying DevOpsDays tickets to our attendees + vendors + speakers. The registration team needs to collaborate and work with other organizer teams (specifically Website, Swag, and Volunteers) since multiple areas are tightly coupled with the registration organizer team.
 
 ### General Tasks
 
 Can you name typical tasks? For example:
```
Comment (nimbinatus) [inline registration_runbook.md:12]:
```suggestion
```
This line was boilerplate :)

File: registration_runbook.md [hunk 2]
```diff
@@ -1,57 +1,42 @@
...
 
-* Write a compelling CFP (the CFP committee)
-* Requisition a unique annual logo (swag)
-* etc.
+* Setup the Ticket Registratiation Tool (Verbage, Finalize Ticket Prices, Vendor Codes, etc.).
+* Work with Website Organizer Team to coordinate when we should promote ticket sales (Early Bird, Normal Sale, etc.).
```
Comment (nimbinatus) [inline registration_runbook.md:14]: Dates (the when) is a core question, but you are coordinating with website to get the links up

File: registration_runbook.md [hunk 3]
```diff
@@ -1,57 +1,42 @@
...
+* Work with Swag Organizer Team to ensure we have enough t-shirts for the total number of attendees.
```
Comment (nimbinatus) [inline registration_runbook.md:15]:
```suggestion
* Work with Swag Organizer Team to ensure we have enough t-shirts, lanyards, and badges for the total number of attendees.
```

File: registration_runbook.md [hunk 4]
```diff
@@ -1,57 +1,42 @@
...
-* Get on stage.
+### After The Conference
+* May / June 2024 - SurveyMonkey emails to collect various forms of feedback (Attendee, Volunteer, Organizer, etc.) 
 
 ## Reminders or Common Mistakes
 
-What do you always try to remember when you come back to this every year? Do you often find yourself scrambling for vendors? Are you always trying to remember where you left your towel? Leave notes for the next person so we can improve :)
-
-## Refund Requests
+**Important Note: Prior to 2024 we used Eventbrite (2024 will be the first year we are using X tool), so keep in mind some of the common mistakes may be framed around using the Eventbrite tool.**
```
Comment (nimbinatus) [inline registration_runbook.md:37]:
```suggestion
**Important Note: Prior to 2024 we used Eventbrite (2024 will be the first year we are using the Pretix tool), so keep in mind some of the common mistakes may be framed around using the Eventbrite tool.**
```

**Review by nimbinatus — CHANGES_REQUESTED** [review]
Couple small changes, but this is a great start! Lmk when you address these comments and suggestions, and I'll merge it.

**Comment by moutons** [comment]
Great start! I would like to see more information about prep-day and day-of responsibilities, broken down between what the Organizers and Volunteers who are doing this work are responsible for. It doesn't necessarily need to be an explicit list of every individual task that needs to happen during the conference, but it would be good to note what the group is responsible for in order to pull off the conference smoothly ("don't forget this" sort of stuff) and where teams need to work together, for example:

Prior to the conference:
- On prep day, set up registration tables, daily swag, badges, and related equipment (Registration Orgs, Volunteers, Swag Orgs)
- On prep day, make sure badges are easily distributed as people register

During the conference:
- Distribute shirts and daily swag as people register (Volunteers)

Thanks

File: registration_runbook.md (hunk 2 above)
Comment (moutons) [inline registration_runbook.md:12]:
```suggestion
* Work with Website and Core Organizer Team to coordinate when we should promote ticket sales (Early Bird, Normal Sale, etc.).
```

**Review by nimbinatus — APPROVED** [review]
approving knowing #20 is out there

**Pull request merged**